test/integration/src/peggy2/test_peggy2_wrong_signature.py

```diff
@@ -0,0 +1,30 @@
+import siftool_path
```

Comment:
Maybe you want to add this to `execute_integration_tests_peggy2.sh`?

Reply:
sure, will add it in execute shell.

Reply:
sure, will add it.